src/web/sections/portfolio/project.astro

```diff
@@ -18,24 +18,32 @@ const alt = `${name} project thumbnail`;
 <div class="flex flex-col gap-6">
   <div class="flex flex-col gap-4">
     <div class="flex gap-6">
-      <Photo
-        src={image}
-        alt={alt}
-        width={200}
-        height={200}
-        class="rounded-lg object-cover max-w-[120px] overflow-hidden sm:block hidden"
-      />
       <div class="flex w-full flex-col gap-4">
-        <div class="flex justify-between">
-          <div>
-            <Typography variant="item-title">{name}</Typography>
-            <Timestamp dates={dates} />
-          </div>
-          <div class="flex gap-2">
-            {links.map((link) => <LinkButton {...link} />)}
+        <div class="flex gap-4">
+          {
+            image && (
+              <div class="hidden flex-1 sm:block">
```

Comment:
Czy tych stylów nie możemy ustawić bezpośrednio na zdjęciu. Niby to detal, ale Lighthouse już teraz marudzi, że mamy za dużo elementów na stronie 😅